SecretSanta/src/SecretSanta.Api/SecretSanta.Api.csproj

```
@@ -8,6 +8,12 @@
   <ItemGroup>
     <Compile Include="..\..\GlobalSuppression.cs" Link="GlobalSuppression.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.0.0" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.0.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0">
```

```suggestion
    <PackageReference Include="coverlet.collector" Version="1.2.0"/>
```

1>Build FAILED.
1>
1>...\SecretSanta\src\SecretSanta.Api\SecretSanta.Api.csproj(16,5): error MSB4025: The project file could not be loaded. The 'PackageReference' start tag on line 15 position 6 does not match the end tag of 'ItemGroup'. Line 16, position 5.
1>    0 Warning(s)
1>    1 Error(s)
1>
1>Time Elapsed 00:00:00.13

---

Darn! Good eyes and thank you for the tip on the diag settings!

---

Yeah np! I can't figure out how to get the .dll message to go away, I've tried cleaning the solution and deleting the bin file to no avail. Not going to mess with it too much more, but I would be curious to hear how you get it to work, as I see it's still failing. ):

---

So... I used your suggestion and changed the setting. Still failed on me but for some reason I didn't see any warnings, again. Then after I realized it failed I went back in and VS threw an error at me in the Program.cs file. I realized I had not removed the Intellitect Analyzers references. I cleaned up some proj file refs, did a clean/rebuild, and something in there resolved my issue. I'm pretty sure I missed removing those analyzers somewhere along the way.

---

Ah okay! Glad you fixed it. Good to know, thanks for the update! :)